src/utils/searchQuery.js

```diff
@@ -0,0 +1,13 @@
+const getSearchQuery = (params) => {
+  let query = ''
+  params.forEach(param => {
+    Object.keys(param).forEach(key => {
+      const value = encodeURIComponent(`${param[key]}`)
+      query += `&${key}=${value}`
+    })
+  })
+  
+  return query
+}
```

Comment:
this could be improved:

```suggestion
const getSearchQuery = (params) => {
  const query = Object
    .entries(params)
    .reduce((result, [key, rawValue]) => {
      const value = encodeURIComponent(`${rawValue}`)

      return result + `&${key}=${value}`
    }, '');

  return query
}
```

Reply:
This suggestion looks great, updating it in the next PR. 
Thanks!